annotations/pr.json

```diff
@@ -0,0 +1,36 @@
+{
+    "command": "pr",
+    "cases":
+    [
+    	{
+            "predicate":
+            {
+            	"operator": "and",
+            	"operands":
+            	[
+            	    {
+            		"operator": "exists",
+            		"operands": ["-T, --omit-pagination"],
+            	    },
+            	    {
+            		"operator": "!exists",
+            		"operands": ["integer"],
+            		"comment": "Unsure how to express this kind of thing. Want to say: operand does not contain an integer."
```

Comment:
Can you add the word `TODO(@DanielSongShen)` here? And also add this filename and line into the README.md file, in a section called `## Issues` or something? Thanks!